cmd/trtl/main.go

```diff
@@ -118,6 +118,47 @@ func main() {
 				},
 			},
 		},
+		{
+			Name: "db:put",
+			// TODO: would we ever want to put multiple values using the CLI? If so, what arg format would we expect to use?
+			Usage:    "put a single value to a key in the trtl database",
+			Category: "client",
+			Before:   initDBClient,
+			Action:   dbPut,
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:    "replica-endpoint",
+					Aliases: []string{"u"},
+					Usage:   "the url to connect to the trtl replication service",
+					EnvVars: []string{"TRISA_DIRECTORY_REPLICA_URL"},
+				},
+				&cli.StringFlag{
+					Name:    "key",
+					Aliases: []string{"k"},
+					Usage:   "specify the key as a string",
+				},
```

Comment:
I went back and forth on what to call this, since `key` seemed more intuitive, but it would mean a different interface to `Get`